docs/routing.md

```diff
@@ -24,6 +24,13 @@ DNS resolution of host names is external to the OpenShift system.
 Please note, that starting the router in the vagrant environment requires it to be pulled into docker.  This may take some time.
 Once it is pulled it will start and be visible in the `docker ps` list of containers and your pod will be marked as running.
 
+A router requires a service account that has access to a security context constraint which allows host ports.
```

Comment:
added same command from https://github.com/openshift/origin/pull/3904 to display SCCs to that support host ports